syntax = "proto3";
package task;





/*
GetIdleHosts返回
*/
message GetIdleHostsResponse {
    /*
    闲置主机Id列表
    */
    repeated string idleHosts = 1 ;
}

/*
GetIdleHostsApi返回
*/
message GetIdleHostsResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    task.GetIdleHostsResponse data = 4 ;
}

